bears/xml2/XMLBear.py

```diff
@@ -21,13 +21,27 @@ def path_or_url(xml_dtd):
         return path(xml_dtd)
 
 
+def xml_style_valid(xml_style):
+    """
+    Checks if xml_style is valid.
+    :param xml_style: Setting key.
+    :return:          Returns the value if valid else returns none.
+    """
+    style = str(xml_style)
+    if style not in XMLBear._styles:
+        logging.warn('Unrecognised style ' + style + '. Valid xml'
+                     ' styles are c14n, c14n11, exc-c14n and oldxml10.'
```

Comment:
ideally coala/coala#3486 should be solved first, that'll save us some hassle later